crates/astria-conductor/tests/blackbox/helpers/macros.rs

```diff
@@ -122,16 +123,31 @@ macro_rules! mount_celestia_blobs {
                 $celestia_height,
                 $crate::sequencer_namespace(),
                 vec![blobs.header],
+                $delay,
             )
             .await;
         $test_env
             .mount_celestia_blob_get_all(
                 $celestia_height,
                 $crate::rollup_namespace(),
                 vec![blobs.rollup],
+                $delay,
             )
             .await
     }};
+    (
+        $test_env:ident,
```

Comment:
I'd flip this and put the special case as the first match, and the base case (which includes the `$delay` pattern) as the last arm of the pattern match.